runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsPlugin.java

```diff
@@ -166,4 +189,80 @@ public void onGameStateChanged(GameStateChanged event)
 			ladders.clear();
 		}
 	}
+
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.BARROWS_REWARD_GROUP_ID && config.showChestValue())
+		{
+			ItemContainer barrowsRewardContainer = client.getItemContainer(InventoryID.BARROWS_REWARD);
+			List<Integer> itemIds = new ArrayList<>();
+			Map<Integer, Item> itemMap = new HashMap<>();
+			chestPrice = 0;
+
+			for (Item item : barrowsRewardContainer.getItems())
+			{
+				if (item.getId() != -1)
+				{
+					if (item.getId() == COINS_995)
+					{
+						chestPrice += item.getQuantity();
+						continue;
+					}
+					itemIds.add(item.getId());
+					itemMap.put(item.getId(), item);
+				}
+			}
+
+			CompletableFuture<ItemPrice[]> future = itemManager.getItemPriceBatch(itemIds);
+			future.whenComplete((ItemPrice[] itemPrices, Throwable ex) ->
+			{
+				if (ex != null)
+				{
+					log.debug("Error looking up item prices", ex);
+					return;
+				}
+
+				if (itemPrices == null)
+				{
+					log.debug("Error looking up item prices");
+					return;
+				}
+
+				log.debug("Price lookup is complete. {} prices.", itemPrices.length);
+
+				try
+				{
+					for (ItemPrice itemPrice : itemPrices)
+					{
+						if (itemPrice.getItem() == null)
+						{
+							continue; // cached no price
+						}
+
+						long itemStack = (long) itemPrice.getPrice() * (long) itemMap.get(itemPrice.getItem().getId()).getQuantity();
+						chestPrice += itemStack;
+					}
+				}
+				catch (Exception ex2)
+				{
+					log.warn("error calculating price", ex2);
+				}
+				finally
+				{
+					final ChatMessageBuilder message = new ChatMessageBuilder()
```

Comment:
I'm not sure why this is in a `finally` like this.. it should be in the `try`. I think the intent of the `try` was to log exceptions since generally if you throw them out of lambdas they get eaten.